framework/doc/content/syntax/Executioner/FixedPointAlgorithms/index.md

```diff
@@ -0,0 +1,61 @@
+When a MultiApp is a sub-app and the main application of its own sub-apps, MOOSE allows relaxation of the MultiApp solution
+within the main coupling iterations and within the coupling iterations, where the MultiApp is the main app, independently.
+
+Relaxation, or acceleration (cf secant/Steffensen's method), is performed on variables or postprocessors. These two objects encompass
+most of the data transfers that are performed when coupling several applications.
+
+## Picard fixed point iterations
+
+Picard iterations are the default fixed point iteration algorithm. They may be relaxed, with a relaxation factor specified for the
+,ain application in the `Executioner` block, and a relaxation factor specified for each `MultiApp` in their respective block.
+
+Relaxed Picard fixed point iterations may be described by:
+\begin{equation}
+x_{n+1} = \alpha f(x_n) + (1 - \alpha) x_n
+\end{equation}
+with $x_n$ the specified variable/postprocessor, $f$ a function representing the coupled problem and $\alpha$ the relaxation factor.
+
+Convergence of Picard iterations is expected to be linear when it converges. The Picard-Lindelhof theorem provides a set of conditions under
+which convergence is guaranteed.
+
+## Secant method
+
+The secant method is a root finding technique which follows secant lines to find the roots of f. It is adapted here for fixed point iterations.
+The secant method may be described by:
+\begin{eqnarray}
+y_{n} = f(x_{n})
+x_{n+1} = x_n - \dfrac{(f(x_n) - x_n) * (x_n - x_{n-1})}{f(x_n) - x_n - f(x_{n-1}) + x_{n-1}}
+\end{eqnarray}
+
+Convergence of the secant method is expected to be super-linear when it converges, with an order of $\dfrac{1 + \sqrt{5}}{2}. Some conditions
+for this convergence rate is that the equations are twice differentiable in their inputs, with a fixed point multiplicity of one. Oscillatory
+functions and poor initial guesses can prevent convergence.
+
+## Steffensen's method
+
+Steffensen's method is a root finding technique based on perturbating a solution at a given point to approximate the local derivative. The update is
+then similar to Newton's method which uses the exact derivative.
+
+\begin{eqnarray}
+g(x_{n}) = \dfrac{f(x_{n} + f(x_{n}))}{f(x_{n})}
+x_{n+1} = x_n - \dfrac{f(x_n)}{g(x_{n})}
+\end{eqnarray}
+
+Convergence of Steffensen's method is expected to be quadratic when it converges. However because it requires two evaluations of the coupled
+problem before computing the next term, this method is expected to be slower than the secant method. A poor initial guesses can also prevent convergence.
+
+!alert note
+When using the secant or Steffensen's methods, only specify variables and postprocessors from either the main application or the sub-applications to be accelerated. Specifying variables or postprocessors to be transformed in both applications will not provide as much acceleration, due to the current implementation of the methods. Future work may remove this limitation.
```

Comment:
transformed?

Reply:
transformed is used inside the code. Let s just say accelerated here